tests/unittests/Foundation/BlockClassTests.mm

```diff
@@ -14,6 +14,8 @@
 //
 //******************************************************************************
 
+#ifdef _WIN32
```

Comment:
Moved this to the WindowsOnly dir - I did use at least one instance of WINOBJC somewhere though.

---
In reply to: [70348439](https://github.com/Microsoft/WinObjC/pull/634#discussion_r70348439) [](ancestors = 70348439)